pkgs/data/icons/candy-icons/default.nix

```
@@ -0,0 +1,31 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "candy-icons";
+  version = "2023-10-03";
+
+  src = fetchFromGitHub {
+    owner = "EliverLara";
+    repo = "candy-icons";
+    rev = "bcf24f3308cc5e39f3e7d1bb43cb26f627f9ba8f";
+    sha256 = "sha256-NCXxH04GvrPE5mQO7JrU6SkjA+BaZokhyjKsm8m9p3g=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/icons/candy-icons
+    cp -r . $out/share/icons/candy-icons
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall
    
    mkdir -p $out/share/icons/candy-icons
    cp -r . $out/share/icons/candy-icons
    
    runHook postInstall
  '';
```
See: https://nixos.org/manual/nixpkgs/unstable/#sec-stdenv-phases